draft-ietf-scitt-architecture.md

```diff
@@ -199,7 +199,7 @@ Artifact:
 
 Auditor:
 
-: an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service.
+: an entity that can review entries of a Transparency Service's Append-only Log to prove the application and enforcment of Registration Policy during each Registration process step.
```

Comment:
```suggestion
: an entity that confirms the correctness and consistency of the Transparency Service's Append-only Log, or that the Transparency Service policies have been applied consistently.
```

Reply:
Auditor checks proofs, and ensures business processes. Registration policies are a business process, and are out of scope.

Reply:
I agree with this description of the auditor role.

This is a very different role from the "Gatekeeper" role that approves materials to be included in the Registry, based on registration policies.

Reply:
This is also okay. Which one do we use?